src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react';
+import './Home.css';
+import { Header, Sidebar, VideoCard } from '../../components';
+import axios from 'axios';
+export const Home = () => {
+    const [videos, setVideos] = useState([]);
```

Comment:
We should use context for videos state. Because we need the data in many components & pages

Reply:
thanks for the suggestion i will move it to context